### PR TITLE
Added admin script to allow running admin from npm

### PR DESCRIPTION
**Pull request by brynyShift72**
AB#6614 shift72-template version, if --admin run first it doesn't create ./output/.kibble which it needs so added mkdir before hand.

### CODE REVIEW

**Comment by PeculiarGoat** [comment]
You gonna change this to Shannon's approach from the core-template PR?

**Comment by brynyShift72** [comment]
Yup, just waiting on his merge to my branch in core, then will copy over

**Comment by brynyShift72** [comment]
@PeculiarGoat Should be ready now

**Review by PeculiarGoat — APPROVED** [review]
LGTM!  :+1:

File: package.json [hunk 1]
```diff
@@ -15,9 +15,11 @@
     "js:watch:local": "chokidar local/site/static/js -c 'node tools/copy-js'",
     "js:build": "NODE_ENV=production rollup -c",
     "kibble": "cd output && kibble render",
+    "kibble:start": "mkdir -p ./output/.kibble && test -n \"$npm_config_admin\" && npm run kibble:watch -- --admin || npm run kibble:watch",
     "kibble:watch": "cd output && kibble render --watch --port 8081",
+    "kibble:watch-admin": "cd output && kibble render --watch --admin --port 8081",
```
Comment (l0ud0gg) [inline package.json]: think this line can be nuked

**Review by l0ud0gg — APPROVED** [review]
looks good

**Review by curtisadams28 — APPROVED** [review]
LGTM

**Pull request merged**